dsl_parser/tests/test_outputs.py

```diff
@@ -138,7 +138,8 @@ def test_valid_get_secret(self):
         func = functions.parse(outputs['port']['value'])
         self.assertTrue(isinstance(func, functions.GetSecret))
```

Comment:
There are no changes in capabilities test?

Reply:
probably no reason, get_capability is a runtime func anyway